src/app/store.ts

```diff
@@ -1,10 +1,15 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
-// eslint-disable-next-line import/no-cycle
-import counterReducer from '../features/counter/counterSlice';
+import { usersReducer } from '../features/usersSlice';
+import { postsReducer } from '../features/postsSlice';
+import { commentsReducer } from '../features/comentsSlice';
```

Comment:
The import statement for `commentsReducer` seems to be from a misspelled file path: '../features/comentsSlice'. Verify that the file path is correct and matches the actual file name to prevent import errors.